docker/Makefile

```diff
@@ -24,7 +24,7 @@ run:
 	-dt \
 	-v `pwd`/..:/petastorm \
 	--name petastorm \
-	petastorm
+	petastorm_ci:latest
```

Comment:
Should this be `selitvin/petastorm_ci:latest`?

Reply:
Removed from this PR